vuepress/docs/next/docs/curate/ecr-bundle-details.md

```diff
@@ -1,5 +1,6 @@
 ---
 sidebarDepth: 2
+redirectFrom: /v6.3.2/docs/ecr/ecr-bundle-details.html
```

Comment:
Are we doing redirects for some documents from 6.3.2 to next?

Reply:
mistake!